mlflow/server/js/src/model-registry/components/ModelPage.enzyme.test.tsx

```diff
@@ -20,19 +20,20 @@ import { ModelRegistryRoutes } from '../routes';
 import { ErrorWrapper } from '../../common/utils/ErrorWrapper';
 
 describe('ModelPage', () => {
-  let wrapper;
+  let wrapper: any;
```

Comment:
guessing the type here but ideally if we're adding types they should be correct types
```suggestion
  let wrapper: React.ReactNode;
```